lib/clock.rb

```diff
@@ -0,0 +1,16 @@
+require "clockwork"
+require "delayed_job"
+require "delayed_job_active_record"
+
+require_relative "../app"
+require_relative "../app/jobs/fetch_feed_job"
+require_relative "../app/repositories/feed_repository"
+
+include Clockwork
+
+fetch_interval = (ENV["FETCH_INTERVAL"] || 60).to_i
```

Comment:
Fetching new stories every minute might be too often? What do we think about increasing the default interval to something like once every 10/30/60 minutes (as stated in `docs/VPS.md`/-/`docs/OpenShift.md` respectively)?